sherpa/utils/tests/test_logging.py

```diff
@@ -0,0 +1,37 @@
+#
+#  Copyright (C) 2020  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+import pytest
+import logging
+import numpy as np
+import sherpa
+from sherpa.utils.logging import config_logger, SherpaVerbosity
+from sherpa.astro import ui
+
+logger = logging.getLogger('sherpa.some_module')
+
+def test_logging_verbosity_contextmanager(caplog):
+    logger.warning('1: should be seen')
+    assert len(caplog.records) == 1
+
+    with SherpaVerbosity('ERROR'):
```

Comment:
Ideally I'd like to also see a test with `SherpaVerbosity(logging.ERROR)`  as well (I know that we're just passing on responsibility to `setLevel` so we should be fine, but just to make sure).

Reply:
We can add as many tests as we like, but this one makes on sense to me. That's the responsibility of Python itself. I'm already using the most advanved input method (a string). `logging.ERROR` is literally just an int.